StellarDotnetSdk.Tests/ScValTest.cs

```diff
@@ -46,11 +46,11 @@ public void TestSCWasmVmError()
     {
         var arithDomainError = new SCWasmVmError
         {
-            Code = xdrSDK.SCErrorCode.SCErrorCodeEnum.SCEC_ARITH_DOMAIN,
+            Code = xdrSDK.SCErrorCode.SCErrorCodeEnum.SCEC_ARITH_DOMAIN
```

Comment:
Let's have a shot call on how we can share rider formatter settings this is something I've changed and choose to save team wide. But appearantly that is not working.. Now you undo formatting I've added the other day.